rfc6979.go

```diff
@@ -23,9 +23,17 @@ import (
 )
 
 // mac returns an HMAC of the given key and message.
-func mac(alg func() hash.Hash, k, m, buf []byte) []byte {
+func mac(alg func() hash.Hash, k, m, buf []byte) ([]byte, hash.Hash) {
```

Comment:
Documentation can be adjusted then.

Reply:
It strangely still fits, because it's still HMAC, just two kinds of it. But updated anyway.